roles/network-multus/defaults/main.yml

```diff
@@ -0,0 +1,41 @@
+namespace: "kube-system"
+cluster: "openshift"
+apb_action: "provision"
+
+multus_provisioner_repo: "docker.io/nfvpe/multus"
+multus_provisioner_release: "latest"
+
+cni_provisioner_repo: "quay.io/schseba/cni-plugins"
+cni_provisioner_release: "latest"
+
+l2_cni_plugin_repo : "quay.io/schseba/l2-bridge-cni-plugin"
```

Comment:
could you please also deploy ovs-cni? https://quay.io/repository/kubevirt/ovs-cni-plugin

Reply:
@leongold Have a PR for that #371